docs/guides/1-step-by-step-guide.md

```diff
@@ -2,15 +2,15 @@
 
 This guide will walk you through all the steps to quickly setup and get started with Yarr.  However, if you want to check out the code of the completed guide first, you can do so here ⬇️.
 
-[![Open in StackBlitz](https://developer.stackblitz.com/img/open_in_stackblitz.svg)](https://stackblitz.com/edit/react-fk8j42?file=src/App.js)
+[![Open in StackBlitz](https://developer.stackblitz.com/img/open_in_stackblitz.svg)](https://stackblitz.com/edit/react-fk8j42?file=src/App.tsx)
 
 
 
 ## Step 1. Create React App
 
 For this guide, we'll start with a create-react-app from here ⬇️. 
 
-[![Open in StackBlitz](https://developer.stackblitz.com/img/open_in_stackblitz.svg)](https://stackblitz.com/edit/react-fk8j42?embed=1&file=src/App.js)
+[![Open in StackBlitz](https://developer.stackblitz.com/img/open_in_stackblitz.svg)](https://stackblitz.com/edit/react-jbqeqg?file=src/App.tsx)
```

Comment:
This link 404s for me

Reply:
@Joseph Strange. I am able to load these links even on incognito.  :thinking_face: